cmd/operator/kodata/knative-eventing/1.7.7/1-eventing-crds.yaml

```diff
@@ -0,0 +1,2296 @@
+# Copyright 2020 The Knative Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  creationTimestamp: null
+  labels:
+    eventing.knative.dev/release: "v1.7.7"
```

Comment:
Is there a way that we can separate actual change from updates?

Reply:
I need to run ./hack/update-codegen.sh to update all the manifests, so that the prow can pass. The prow always checks the match-up between the pr and the expected manifests.